pipeline/workflow/pipeline_utils.py

```diff
@@ -117,6 +119,17 @@ def concatenate_symbols(symbols):
     return concatted_symbols
 
 
+def concatenate_files_with_identical_headers(file_directory, output):
```

Comment:
A minor thing, but in the spirit to keep the code base concise, one could also do (unless there is a subtlety I miss):
```
 subprocess.run(f'cat {file_directory}/*.txt > out', shell=True)
```

Reply:
yea... this function throws out the first line (header row) of every file after the first